ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -272,8 +272,9 @@ StackLayout {
 
             property string communityId
 
-            loginType: root.rootStore.loginType
             walletAccountsModel: WalletStore.RootStore.nonWatchAccounts
+            canProfileProveOwnershipOfProvidedAddressesFn: WalletStore.RootStore.canProfileProveOwnershipOfProvidedAddresses
```

Comment:
We are trying to avoid using `singleton stores` bc it makes the components testing more complicated. However, it's a big topic and proper refactor pending for it, but just FYI if you can avoid it the most possible, it'd be fantastic!

Reply:
That would be next step.